ansible/roles_ocp_workloads/ocp4_workload_cloud_native_robot/templates/apps/application.yaml.j2

```diff
@@ -0,0 +1,28 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: applications
+  namespace: {{ _ocp4_workload_cloud_native_robot_shared_namespace }}
+  finalizers:
+  - resources-finalizer.argocd.argoproj.io
+spec:
+  destination:
+    name: ''
+    namespace: {{ _ocp4_workload_cloud_native_robot_shared_namespace }}
+    server: 'https://kubernetes.default.svc'
+  project: default
+  syncPolicy:
+    automated:
+      prune: false
+      selfHeal: false
+  source:
+    repoURL: http://{{ ocp4_workload_cloud_native_robot_shared_gitea_instance }}.{{ ocp4_workload_cloud_native_robot_shared_gitea_project }}.svc:3000/{{ _ocp4_workload_cloud_native_robot_shared_user }}/{{ ocp4_workload_cloud_native_robot_shared_gitea_repo }}
+    targetRevision: main
```

Comment:
same? this should be a tag   and a variable so it can be overwritten if needed.

Reply:
Thanks, I removed the file. Because it's not in use. (copy&paste)...